config/default.yml

```diff
@@ -1,10 +1,10 @@
 ---
 RSpec:
   Enabled: true
-  Include:
+  Include: &1
```

Comment:
It seems that this could be named, e.g. alphanumeric. Might make the file less scary, if it has proper names :)

Reply:
I vaguely recall I planned to do so (to mimic the well-known `&default` from `config/database.yml` 😄), but ended up with numbers somehow. Fixed 👍

Reply:
Ah, now I recall. It's `bin/build_config` that does that:
```diff
--- a/config/default.yml
+++ b/config/default.yml
@@ -1,10 +1,10 @@
 ---
 RSpec:
   Enabled: true
-  Include: &include
+  Include: &1
     - "**/*_spec.rb"
     - "**/spec/**/*"
-  Language: &language
+  Language: &2
     inherit_mode:
       merge:
         - Expectations
```

Supposedly, `YAML.dump(unified_config)` has no respect for named anchors.

I'll revert, otherwise CI is red.